### PR TITLE
feat(series): add native support for next episodes

### PR DESCRIPTION
**Pull request by AntonLantukh**
## Description

- `after_id` query param is added
-  complex front-end related logic is removed

### Steps completed:

According to our definition of done, I have completed the following steps:

- [ ] Acceptance criteria met
- [ ] Unit tests added
- [ ] Docs updated (including config and env variables)
- [ ] Translations added
- [ ] UX tested
- [ ] Browsers / platforms tested
- [ ] Rebased & ready to merge without conflicts
- [ ] Reviewed own code

### CODE REVIEW

File: src/services/api.service.ts [hunk 1]
```diff
@@ -149,13 +149,20 @@ export const getSeriesByMediaIds = async (mediaIds: string[]): Promise<{ [mediaI
  * Get all episodes of the selected series (when no particular season is selected or when episodes are attached to series)
  * @param {string} seriesId
  */
-export const getEpisodes = async (seriesId: string | undefined, pageOffset: number, pageLimit?: number): Promise<EpisodesWithPagination> => {
+export const getEpisodes = async (
+  seriesId: string | undefined | null,
+  pageOffset: number | null,
+  pageLimit: number | null = PAGE_LIMIT,
+  afterId?: string | null,
```
Comment (budzworthy) [inline src/services/api.service.ts]: I always like to use named params when there are this many inputs of native types to a function

File: src/services/api.service.ts [hunk 2]
```diff
@@ -149,13 +149,20 @@ export const getSeriesByMediaIds = async (mediaIds: string[]): Promise<{ [mediaI
...
+): Promise<EpisodesWithPagination> => {
   if (!seriesId) {
     throw new Error('Series ID is required');
+  } else if (pageOffset && afterId) {
```
Comment (budzworthy) [inline src/services/api.service.ts]: I would suggest we either let the BE handle this or enforce it with typings on the arguments:
```
args: {seriesdId: string} & ({pageOffset: number, pageLimit: number} | {afterId: string})
```

File: src/services/api.service.ts (hunk 1 above)
Comment (AntonLantukh) [reply src/services/api.service.ts]: Was thinking about it, but wanted to follow the approach we already have for other endpoint. Changed it for episodes and seasons endpoints.

File: src/services/api.service.ts (hunk 2 above)
Comment (AntonLantukh) [reply src/services/api.service.ts]: Removed this check

**Review by budzworthy — APPROVED** [review]

**Comment by AntonLantukh** [comment]
Waiting for back-end related fixes

**Pull request merged**